CRM/Utils/System/Standalone.php

```diff
@@ -387,7 +387,7 @@ public function getCiviSourceStorage(): array {
     }
 
     return [
-      'url' => CRM_Utils_File::addTrailingSlash('', '/'),
+      'url' => CRM_Utils_File::addTrailingSlash(CRM_Core_Config::singleton()->resourceBase, '/'),
```

Comment:
It might be a bit of a loop. The definition of `resourceBase` comes from `userFrameworkResourceURL` (via `MagicMerge`) which in turn comes from `[civicrm.root]/` (via `SettingsManager::getSystemDefaults()`).

If it works, it might be because some part is overridden in a setting file?

For reference, in `CRM_Utils_System_DrupalBase`, they seem to get the value from `CIVICRM_UF_BASEURL` (*which is a lower-level/longer-standing option*). Wonder if that might be better?

Reply:
Yeah, I agree the difference is `civicrm-asset-plugin`.

The formula there `CIVICRM_UF_BASEURL . '/core'` sounds a bit better (*fewer dependencies*).

The formula from `DrupalBase::getCiviSourceStorage()` would be a bit more flexible. (*It would detect other folder names -- different from `core`.*) But OTOH, folks with alternate layouts could configure them via `$civicrm_paths`...  So maybe that complexity isn't needed.